.flake8

```diff
@@ -0,0 +1,19 @@
+[flake8]
+# We disable the following inspections:
+# 1. F541: f-string is missing placeholders (we allow f-strings that don't do
+#    any formatting for consistent looks and for future safety)
+# 2. Inspections incompatible with Black (see https://github.com/psf/black/blob/master/docs/compatible_configs.md#why-those-options-above-1):
+#    E203: whitespace before ':'
+#    W503: line break before binary operator
+extend-ignore: F541,E203,W503
+
+# Maximum number of characters on a single line. Default for black, see:
+# https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length
+max-line-length: 88
+
+inline-quotes = single
+multiline-quotes = """
+docstring-quotes = """
+
+# for snakemake pipeline
+builtins = snakemake
```

Comment:
That's interesting! What is this line doing? Make flake8 check snakemake files specifically, or ignore them? Would be nice to add a comment. The reason is that I'm gonna use Snakemake along with flake8 in my projects too, so would be useful to add something on that into the team-docs repo as well.

Reply:
I added some explanation in a comment. Basically, snakemake provides a variable called `snakemake`, when a script is provided in the rule. That way, the script has access to all the objects defined in the Snakefile and its rule. Flake8 does not recognise this, so I specified for it to ignore the `snakemake` variable.